File: Bcore/src/main/java/top/niunaijun/blackbox/http/controllers/TestController.java
package top.niunaijun.blackbox.http.controllers;

import com.yanzhenjie.andserver.annotation.GetMapping;
import com.yanzhenjie.andserver.annotation.RestController;

@RestController
public class TestController {
    @GetMapping("/test")
    public String test()
    {
        return "Hello World";
    }
}

File: app/src/main/java/top/niunaijun/blackboxa/http/controllers/TestController.kt
package top.niunaijun.blackboxa.http.controllers

import com.yanzhenjie.andserver.annotation.GetMapping
import com.yanzhenjie.andserver.annotation.PostMapping
import com.yanzhenjie.andserver.annotation.RequestParam
import com.yanzhenjie.andserver.annotation.RestController

@RestController
class TestController {
    @GetMapping("/test")
    fun login(): String {
        return "Hello World"
    }
}